roles/libvirt_manager/vars/main.yml

```diff
@@ -42,3 +42,6 @@ cifmw_libvirt_manager_storage_pool: "cifmw-pool"
 cifmw_libvirt_manager_fixed_networks_defaults:
   - ocpbm
   - ocppr
+
+cifmw_libvirt_manager_no_dnsmasq_nets_defaults:
```

Comment:
Can we not turn this around?
 Have a variable or an additional property where we define networks to enable dnsmasq, and only set up dnsmasq if enabled for the network.

Reply:
Not sure about the benefit of either approach to be fair. With that one here (disallow-list) we have a working thing without any edition for "the cases I know". We can override/append networks using the parameter without `_defaults` so it's easy enough to set up for jobs.